e2e/utils.go

```diff
@@ -43,6 +43,9 @@ const (
 	// Default key and label for Listoptions
 	appKey   = "app"
 	appLabel = "write-data-in-pod"
+
+	// vaultToken KMS type
+	vaultToken = "vaulttokens"
```

Comment:
`vaultToken` key value can also be set as `vaulttoken` instead of `vaulttokens` ?

Reply:
the value cannot be changed as `vaulttokens` is a standard key we expect in the storageclass.